libc-bottom-half/headers/public/wasi/api.h

```diff
@@ -1,6 +1,6 @@
 /**
  * THIS FILE IS AUTO-GENERATED from the following files:
- *   typenames.witx, wasi_snapshot_preview1.witx
+ *   wasi_snapshot_preview1.witx
```

Comment:
At I see the boilerplate message already doesn't say the tool name, maybe fine to leave that as a followup.